### ISSUE TITLE
clarify network usage

### ISSUE THREAD

**Issue by iqualfragile** [spacejam/void#2]
for some reason this application builds a connection to ipinfo.io, querrying http://ipinfo.io/loc thereby determining the location.
can you explain why this is done, and disable this from happening by default?
is is bad to do a network connection without explicit confirmation.
also this operation blocks/delays startup.

**Comment by spacejam** [comment]
This is done so that I can eventually plot roughly where I was when I was working on various ideas, but I agree that it should be more explicit and easily disabled.  I'll think about a clean way to do this, but for now one hack to make it fail fast is adding this line to `/etc/hosts`

```
:: ipinfo.io
```

**Comment by iqualfragile** [comment]
or just removing most of meta.rs, which also has the advantage of still being able ot use ipinfo.io. its open source after all.
it would be better to use geoclue for this functionality instead of relying on a single website.

**Comment by spacejam** [comment]
Meta will eventually be fed into a columnar in-memory store for rapid, highly specifiable lookups. Right now all children of a particular view are scanned in an O(n) sweep, for every frame that is drawn.  This works fine for the thousand or so nodes I've got in my personal keeper, but this will change when it leads to flickering screens.

I've made location queries explicitly opt-in, by setting the `LOCATION_QUERY` env var. This will continue to be smoothed out over time.

Thanks for the issue!
